### DISCUSSION TITLE
[shape] Praxis Conscious Food Outreach

### DISCUSSION THREAD

**Discussion by laurenwigmore** [life-itself/community#1015] [Ideas]
After an initial meeting with Valerie related to her role as Tenzo, and the knowledge and practices cultivated throughout her years of work. We spoke about how to provide an entry point for individuals who are interested in starting to gain her knowledge around 'conscious food' but who are unable to attend a specific residency/retreat. 

## Acceptance
- [ ] One entry level free resource is available for LI members

## Tasks
- [ ] Brainstorm an online course re collective wisdom with food / conscious food

### Meeting 2023-08-03
- The aim is to open a space that doesn't exist currently for people to connect to food in a space that links all of the practices currently in place for food 
- To help people to connect and think differently through food 
- The real connection comes from people who arrive on-site work and get interested in Valeria's practices not necessarily when people are attending workshops specifically
- Lots of cooks in retreat type spaces are not traditional chefs/cooks - but volunteers who take on that role within their community. This can create stress, overwhelm, which some of the knowledge can help reduce.
- She is also doing/has research with people who eat this way for a week - 'feel more nourished with less'

Future aim to feed this into some form of revenue creation. 
Valerie already has an audience who contact her from her own personal website, these are frequently from people internationally who can't join
Valerie has recently written a book

Future brainstorm could lead into a step-by-step and group calls/dynamic online course

Conscious food alliance - calling for curriculum - open to conscious food practices if already in practice. Future proposal that may need Comms support related: https://consciousfoodsystems.org/call-for-proposals/

### Poll 
Poll in whatsapp group early August - Would you sign up to a free Conscious Food email course? 
Yes - 16 / No - 8 (at 25th Oct 2023)

Notes from Valerie: The name (conscious food) is not being ideal as it implies mindfulness
Instead
**Community Food Design 
For a group transformative effect 
Through specific practices and principles**

### Timeline 
- Early/mid September next available to chat and will send documents across

**Comment by nathenf** [comment]
@laurenwigmore once created tie this into the beacons epic
